### ISSUE TITLE
`--submit-job` can accept general arguments

### ISSUE THREAD

**Issue by kasra-keshavarz** [ShervanGharari/EASYMORE#79]
The `--submit-job` flag can enable accepting more argument for scheduler options. The more users are using this platform, the more flexibility is needed.

**Comment by ShervanGharari** [comment]
Thank you, I think it can be fully removed, I wanted to remove the job from the cli but kept it to accommodate the FHIMP for the time being. I prefer no job call from inside the package. user decides everything outside and calls easymore scripts, config, or cli.

**Comment by kasra-keshavarz** [comment]
So, the script will not automatically submit anything to schedulers in newer versions, is this correct?

**Comment by ShervanGharari** [comment]
Yes. The best option from my point of view would be to make the package separate from the job. I'll remove this in the future version as soon as the dependency of the FHIMP repo is resolved.

**Comment by ShervanGharari** [comment]
@kasra-keshavarz 
Hi Kasra,
I'm revisiting this issue as part of a cleanup. I'm planning to remove the [file](https://github.com/ShervanGharari/EASYMORE/blob/main/src/easymore/scripts/assets/default.slurm) for submitting jobs from EASYMORE. Before doing so, I wanted to check if this would impact any current workflows. If it's still needed, I'm happy to leave it in place temporarily so that other parts of the workflow can be adjusted accordingly.
